README.md

```diff
@@ -320,7 +320,7 @@ MethodSpec beyond = MethodSpec.methodBuilder("beyond")
     .addStatement("result.add($T.createNimbus(\"2001\"))", hoverboard)
     .addStatement("result.add($T.createNimbus($T.THUNDERBOLT))", hoverboard, namedBoards)
     .addStatement("$T.sort(result)", Collections.class)
-    .addStatement("return result.isEmpty() $T.emptyList() : result", Collections.class)
+    .addStatement("return result.isEmpty() ? $T.emptyList() : result", Collections.class)
```

Comment:
More importantly, I think this `isEmpty` check is useless (technically, in the generated code it definitely is; but it doesn't even add anything to the example either). I suggest replacing the line with:
```java
    .addStatement("return result")
```